classes/Form.php

```diff
@@ -402,6 +402,9 @@ public function uploadFiles()
             (array) $settings,
             ['name' => $post['name']]
         );
+        // Allow plugins to adapt settings for a given post name
+        // Useful if schema retrieval is not an option, e.g. dynamically created forms
+        $grav->fireEvent('onFormUploadSettings', ['settings' => $settings, 'post' => $post]);
```

Comment:
OK, so settings in an stdClass object (passed by reference), but how about the `$post`? Should we allow changes in the content and thus, pass it as a reference?